packages/ui-components/src/index.js

```diff
@@ -71,7 +71,7 @@ export { PanelBody } from "./components/PanelBody/index.js"
 export { PanelFooter } from "./components/PanelFooter/index.js"
 export { PageFooter } from "./components/PageFooter/index.js"
 export { PageHeader } from "./components/PageHeader/index.js"
-export { Pagination } from "./components/Pagination/index.js"
+export { Pagination } from "./components/Pagination/Pagination.component"
```

Comment:
**(Breaking change)**
Since component on portal depended I would still export it from deprecated